neurovault/urls.py

```diff
@@ -400,6 +410,22 @@ def images(self, request, pk=None):
             page, context={'request': request}, many=True)
         return paginator.get_paginated_response(serializer.data)
 
+    @detail_route(methods=['post'], url_path='addimage')
```

Comment:
I also haven't figured out yet how to make two "image" `detail_route`s co-exist. Initially I declared the add image endpoint as `POST /collections/<id>/images` with `@detail_route(methods=['post'], url_path='images')` but it turned out to override the `images()` endpoint (one that serves `GET`) for some unknown reason. I may need to study DRF more to figure out why.